src/components/list/_list.scss

```diff
@@ -103,15 +106,13 @@ $include-html: false !default;
     @include mint-list-basic-styles();
     display: inline-block;
     vertical-align: inherit;
-    font-size: $breadcrumbListFontSize;
-    line-height: $breadcrumbListFontSize * 1.5;
     color: $breadcrumbListSeparatorColor;
 
     &__element {
       display: inline-block;
 
       &:after {
-        content: '·';
+        content: '•';
```

Comment:
I have this page from Joanna as a mockup example for VR:

http://typecast.com/QhQQkkddGr/share/87eeaf5d967424f871d1236edad36ee3d46a85b6KrPQgjrbw

I copied the symbol from there directly